src/brc20-bridge/src/lib.rs

```diff
@@ -8,15 +8,19 @@ pub mod memory;
 pub mod ops;
 pub mod state;
 
-use ic_metrics::Metrics;
-
 pub use crate::canister::Brc20Bridge;
 
 const MAINNET_CHAIN_ID: u32 = 0;
 const TESTNET_CHAIN_ID: u32 = 1;
 const REGTEST_CHAIN_ID: u32 = 2;
 
+#[cfg(target_family = "wasm")]
+#[export_candid]
 pub fn idl() -> String {
+    
+    use ic_canister::export_candid;
```

Comment:
If I would be `rustc` I wouldn't compile this, as the macro is used outside of the scope that it is used in...

Maybe just use the full path of the macro instead of importing it? (`#[ic_canister::export_candid]`)